data/api/client-server/report_content.yaml

```diff
@@ -29,11 +29,20 @@ securityDefinitions:
 paths:
   "/rooms/{roomId}/report/{eventId}":
     post:
-      summary: Reports an event as inappropriate.
+      summary: Reports an event as inappropriate. You must be joined to the room in order
+      to report an event.
       description: |-
         Reports an event as inappropriate to the server, which may then notify
         the appropriate people.
```

Comment:
```suggestion
        Reports an event as inappropriate to the server, which may then notify
        the appropriate people. The caller must be joined to the room to report
        it.
```

Reply:
I folded this suggested into the one above; otherwise GitHub gets a bit confused as they overlap.